lib/crud_app/bucket_list.ex

```diff
@@ -5,7 +5,13 @@ defmodule CrudApp.BucketList do
   alias CrudApp.BucketList.Goal
 
   def list_goals do
-    Repo.all(Goal)
+    query = from g in Goal, where: g.is_achieved == false
+    Repo.all(query)
```

Comment:
You could use threading here if you wanted to get a bit more idiomatic.

Reply:
Fixed in f0349f380699be4a1c7612c4f176b1aa91939f48